openedx_webhooks/lib/edx_repo_tools_data/test/models/test_person.py

```diff
@@ -45,7 +45,7 @@ def test_before_expired(self, before_expired_person):
 
     def test_no_agreement(self):
         p = Person('', {'agreement': 'none'})
-        yesterday = arrow.now().replace(days=-1).date()
+        yesterday = arrow.now().shift(days=-1).date()
```

Comment:
Please specify(in the PR description) which dependency upgrade required this change. It is better to document than let people search on their own why this change was made.